lemur/plugins/lemur_digicert/plugin.py

```diff
@@ -215,10 +221,17 @@ def handle_cis_response(response):
     :param response:
     :return:
     """
-    if response.status_code > 399:
-        raise Exception(response.text)
-
-    return response.json()
+    if response.status_code == 404:
+        raise Exception("DigiCert: Order not in issued state.")
+    elif response.status_code == 406:
+        raise Exception("DigiCert: Wrong Header")
```

Comment:
Should it mention about the request format issue, like 'Wrong request format in header' or just 'Wrong request format'

Reply:
good point, not harm in being more explicit.